### ISSUE TITLE
Meshing Workflows - why are old arguments required and is there any advice on how to set these

### ISSUE THREAD

**Issue by spatel-23** [ansys/pyfluent#2414]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Why are old arguments required and is there any advice on how to set these?

Watertight Meshing Workflow
====================
Update boundaries
meshing.workflow.TaskObject["Update Boundaries"].Arguments = {
    "BoundaryLabelList": ["wall-inlet"],
    "BoundaryLabelTypeList": ["wall"],
    "OldBoundaryLabelList": ["wall-inlet"],
    "OldBoundaryLabelTypeList": ["velocity-inlet"],

Fault-Tolerant Meshing Workflow
======================
Update regions settings
        "OldRegionLeakageSizeList": [""],
        "OldRegionMeshMethodList": ["wrap"],
        "OldRegionNameList": ["fluid-region-1"],
        "OldRegionOversetComponenList": ["no"],
        "OldRegionTypeList": ["fluid"],
        "OldRegionVolumeFillList": ["hexcore"]

### 📝 Steps to reproduce

Watertight Meshing Workflow and Fault-Tolerant Meshing Workflow examples show these.

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2024R1

### 🐍 Which Python version are you using?

3.8

### 📦 Installed packages

```shell
NA
```

**Comment by raph-luc** [comment]
My understanding of the basics here is that you need a way to specify which boundaries/regions you are changing when providing the arguments for the task object. 

For example, if your geometry initially has boundary labels `["wall-1", "wall-2", "inlet", "outlet"]`, respectively associated with types `["wall", "wall", "velocity-inlet","pressure-outlet"]`, and you want to change `"wall-1"` **label** to `"mass-inlet"` and its **type** from `"wall"` to `"mass-flow-inlet"`, you'd provide the following:
```
meshing.workflow.TaskObject["Update Boundaries"].Arguments = {
  "BoundaryLabelList": ["mass-inlet"],
  "BoundaryLabelTypeList": ["mass-flow-inlet"],
  "OldBoundaryLabelList": ["wall-1"],
  "OldBoundaryLabelTypeList": ["wall"],
}
```

Similarly, in the Watertight Meshing Workflow example you shared above, those arguments are changing the **type** of the boundary labeled `"wall-inlet"` from `"velocity-inlet"` to `"wall"`.

Does this explanation make sense?

**Comment by spatel-23** [comment]
Thanks for quick answer - that's useful to know and makes sense, as you point out, to specify old values to identify boundaries/regions you want to change. But not sure if it makes sense to also specify the old type. 
The point I am trying to make is that inputting such old arguments can be tedious for large complex cases especially when they don't seem to be needed.

**Comment by raph-luc** [comment]
I fully agree with you. I don't personally like the way this is set up, and I don't know if it really needs to be this way. I will discuss this with the team, thanks for bringing this up.
